plugins/by-name/clangd-extensions/default.nix

```
@@ -76,18 +76,34 @@ lib.nixvim.plugins.mkNeovimPlugin {
 
   extraConfig = cfg: {
     warnings = lib.nixvim.mkWarnings "plugins.clangd-extensions" {
-      when = !config.plugins.lsp.enable;
+      when =
+        !(
+          (config.plugins.lsp.servers.clangd.enable or false) || (config.lsp.servers.clangd.enable or false)
```

```suggestion
          config.plugins.lsp.enable || config.lsp.servers.clangd.enable
```

Maybe we want an additional warning for:

```nix
{
  when = config.lsp.servers.clangd.enable && !config.plugins.lspconfig.enable;
  message = "TODO: you probably want nvim-lspconfig";
}
```

Although, IDK a sane way to suppress such a warning when users know how to configure the server without installing nvim-lspconfig to set defaults for them?

---

Right, again I think we don't want to pester the user in this case.